src/renderer/atlas/AtlasEngine.api.cpp

```diff
@@ -141,6 +141,11 @@ constexpr HRESULT vec2_narrow(U x, U y, AtlasEngine::vec2<T>& out) noexcept
     return S_OK;
 }
 
+[[nodiscard]] HRESULT AtlasEngine::NotifyNewText(const std::wstring_view newText) noexcept
+{
+    return S_OK;

```

Comment:
you added a base implementation that returns `S_FALSE`; does atlas need an `S_OK` version?

Reply:
AtlasEngine doesn't inherit from `RenderEngineBase`.